dev/io.openliberty.data.internal.persistence/src/io/openliberty/data/internal/persistence/cdi/FutureEMBuilder.java

```diff
@@ -150,6 +152,9 @@ EntityManagerBuilder createEMBuilder() {
                                         resourceName, metadataIdentifier, application, module, component, entityTypes);
 
                 } catch (NamingException x) {
+                    FFDCFilter.processException(x, this.getClass().getName() + ".createEMBuilder", "155", this, new Object[] { (switchMetadata ? repoMetadata : extMetadata) });
```

Comment:
FFDC is automatically added, and the other information about which metadata will show in the following exception.
```suggestion
```